src/dash-table/derived/cell/wrappers.tsx

```diff
@@ -28,7 +28,7 @@ class Wrappers {
             (column, columnIndex) => this.getWrapper(
                 false,
                 false,
-                rowIndex,
+                rowIndex + _offset.rows,
```

Comment:
You can rename `_offset` to `offset`. The underscore was there to prevent tslint from flagging it as an unused parameter since it was required for caching purposes but not when calculating.